packages/platform-browser/src/dom/events/hammer_gestures.ts

```diff
@@ -85,7 +85,7 @@ export interface HammerInstance {
 }
 
 /**
- * An injectable [HammerJS Manager](http://hammerjs.github.io/api/#hammer.manager)
+ * An injectable [HammerJS Manager](https://hammerjs.github.io/api/#hammer-manager)
```

Comment:
This doesn't work for me. `#hammermanager` (without the hyphen) does:

```suggestion
 * An injectable [HammerJS Manager](https://hammerjs.github.io/api/#hammermanager)
```

Reply:
Good that you spotted this! I have used the links from the table of contents at the top, but it appears they are broken (https://github.com/hammerjs/hammerjs.github.io/issues/66)